docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+
+    Querying the metadata catalogue for Sentinel2 data
+    Sensor(s): S2B
+    Tile(s): T33UUU,T33UVU
+    Daterange: 20190101 to 20190430
+    Cloud cover minimum: 0%, maximum: 70%
+
+    44 Sentinel2 Level 1 scenes matching criteria found
+    21.06GB data volume found.
+
+    Starting to download 44 Sentinel2 Level 1 scenes
+
+
+    Downloading S2B_MSIL1C_20190119T101339_N0207_R022_T33UVU_20190119T121912(5 of 44)...
+
+    |===========                                                                            |  13 %
+
+
+The status bar will update after every scene that has successfully been downloaded. Any scenes that have already been downloaded will be skipped and a notification will be printed. If the download was canceled at some point and a scene wasn't fully downloaded, FORCE Level 1 CSD will delete that scene and redownload it.
+
+
+A note on Sentinel-2 scenes
+"""""""""""""""""""""""""""
+
+The processing baseline for Landsat data is standardized by the Collection format. For Sentinel-2 we found that there were duplicate scenes with different processing baselines or processing dates. FORCE Level 1 CSD will check for duplicates and only download the version with the highest processing baseline or latest processing date if the processing baseline is the same.
+
+
+Saving metadata
+---------------
+
+FORCE Level 1 CSD offers the possibility to save the metadata of the scenes that matched the specified requirements. Having easy access to this kind of information can be very helpful when creating statistics and visualizations about data availability over time, cloud cover distribution over time, data volume (e.g., per sensor or year), etc. Using the ``-k | --keep-meta`` option will save the results of the current query to the Level 1 datapool folder under the file name
+
+.. code-block:: none
+
+    csd_metadata_YYYY-MM-DDTHH-MM-SS.txt
+
+
+The ``-k | --keep-meta`` option can be combined with the ``-n | --no-act`` option to acquire the metadata without downloading any data.
+If Landsat and Sentinel-2 data is queried at the same time, two separate files will be created. Please note that Google's metadata catalogues use different column names for Landsat and Sentinel-2.
+
+
+Scenes from the same path/row (Landsat) or MGRS tile (Sentinel-2) are stored in folders
+
+
+Information for Docker users
+----------------------------
+
+If you are running FORCE in Docker, there won't be user folder in the home directory. This is a problem for gsutil as the config file is stored there by default. You can work around this by mounting the folder where your ``.boto`` file is located and set an environment variable that points gsutil there. If you're only running gsutil in the docker image, you don't need to mount your local folder. Just set the environment variable ``BOTO_CONFIG`` to wherever you'd like to store the config and run ``gsutil config``.
```

Comment:
I don't advise doing this. Please delete. Do not store secrets in a Docker image

Reply:
Btw, I had a phone call with Erik last week, and we wondered wheter the .boto can be transferred between machines? Do you know if this works?

Reply:
I don't see anything in there that would be specific to a machine so my answer would be yes. 
Just swap them between R2D2 and BB8 and see what happens.
And say Hi to Erik ;)

Reply:
I understand the concern, but how are you going to run `gsutil config` from a system where it is not installed? Users may run docker from machines where they don't have any rights to install gsutil (and probably also no need except for creating the config file)